nntrainer/tensor/manager.h

```diff
@@ -169,6 +169,7 @@ class Manager {
   /**
    * @brief     Create tensors with the given spec
    *
+   * @param w   node Graph node to extract node identifiers/info
```

Comment:
Looks like for some reason, @param w was used everywhere before lol. Could you bother to change this?

Reply:
sure, done.